wikimedia/wikipedia-ios
# Add accessibility label, custom actions to notifications center cell

**Phabricator:**  https://phabricator.wikimedia.org/T296779 and partially addresses https://phabricator.wikimedia.org/T288776 and 

### Notes
* The intention is to make each cell an accessible element, providing context through an accessibility label and making the swipe actions available to VoiceOver users. 
* Note: the idea is for the VoiceOver behavior to be similar to the Apple Mail app.

### Test Steps
1. With VoicerOver on, go to the notifications center
2. You should be able to navigate through the items on the screen by swiping right to left.
3. When selecting a notification, verify the content read. It should be:
- the type of the notification
- its project
- title
- read/unread status 
- Actions available
- Example: `"Talk page message notification. English Wikipedia. Topic title. Read. Actions available"`
4. Tapping twice on a selected notification should take it to it
5. Swiping up and down on the notification should present the actions available: 
- Mark as Read or Unread, 
- More menu
-  Open notification.

6. Tapping twice on the screen should execute the selected action.
Marking items as unread makes a subtle sound as it changes the screen.
7. Then, Select the Edit button.
8. Swipe left to right to navigate through the notifications, and tap twice to select.
9. When selecting an item, you should hear "Selected"
10. The default VoiceOver behavior announces only the selected items
11. Navigate to the bottom bar buttons and make sure you can read and activate them.
12. Make sure you can access the Mark button and its sub-menu
13. Make sure you can, when available, mark all as read

### Screenshots/Videos
